packages/framework/test-client-utils/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@fluidframework/test-client-utils",
+  "version": "0.48.0",
+  "description": "Fluid client development and test utilities",
+  "homepage": "https://fluidframework.com",
+  "repository": "https://github.com/microsoft/FluidFramework",
+  "license": "MIT",
+  "author": "Microsoft and contributors",
+  "sideEffects": false,
+  "main": "dist/index.js",
+  "module": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": "concurrently npm:build:compile npm:lint",
+    "build:compile": "npm run tsc && npm run build:test",
+    "build:full": "npm run build",
+    "build:full:compile": "npm run build:compile",
```

Comment:
need a build:docs task and api-extractor config files, right? Otherwise there are no API docs.

Reply:
Added! A bit confusing how our different packages have different scripts on them. I was basing this off of the test-runtime-utils package which didn't have them, but azure-client does. Added in some of the other missing scripts as well